spec/features/farms/crops/index_spec.rb

```diff
@@ -92,4 +92,23 @@
     expect(current_path).to eq("/farms/#{@farm.id}/crops")
     expect("Peaches").to appear_before("Tomatoes")
   end
+
+  it "can submit a form that filters results" do
+
+    @crop4 = @farm.crops.create!(name: 'Cabbage', yield: 12, annual: true)
```

Comment:
I don't think this crop needs to be an instance variable unless it's in a before :each block